DEV_IDE_SETUP.md

```diff
@@ -0,0 +1,66 @@
+# IntelliJ Development Instance Setup
+
+## Determine which JDK you're running, and where it is
+
+Inside IntelliJ, go to the SDK menu (`Cmd+;` or `Ctrl+;`).
+
+Go to `Project`, and look under `SDK` to find the JDK you intend to use
+to develop the plugin.
+
+Press `Edit`, and make a note of `JDK home path`, as well as the version of the JDK.
+
+## Download and Install the JetBrains Runtime SDK
+
+Head to [the JetBrains Runtime releases page](https://github.com/JetBrains/JetBrainsRuntime/releases)
+and pick a version of the `JBRSDK` that matches your OS, architecture, and JDK version.
+
+Install it appropriately:
+
+- If you've downloaded a tarfile, just note the directory where you untarred it.
+- If you're on macOS and have chosen a `.pkg` release, it will be installed under `/Library/Java/JavaVirtualMachines/jbrsdk-<version and os>/Contents`
+
+Back in IntelliJ, head to the SDK menu (`Cmd+;` or `Ctrl+;`), and go to `SDKs` in the sidebar.
+
+Add a new SDK by pressing the `+` button at the top.
+Pick `Add IntelliJ Platform Plugin SDK`, and select the path where you've just installed it.
+
+Add the java standard library, so that IntelliJ can index common symbols such as `java.io.File`:
+
+- Head to the "Classpath" tab under the IntelliJ Platform SDK you've just created, and press the `+` icon.
+- Navigate to the root directory of the SDK you've noted before (e.g. `/opt/homebrew/Cellar/openjdk@17/17.0.4.1/libexec/openjdk.jdk/Contents/Home` for an openjdk installed via Homebrew).
+- Navigate to `<jdk_home>/Contents/Home/lib/jrt-fs.jar`, and open it.
+- Apply the changes in IntelliJ.
+
+## Import the plugin project.
+
+Clone and import this repository as you would any other Bazel project.
```

Comment:
Can we add instructions about using the existing `.bazelproject` files (e.g. [ijwb/ijwb.bazelproject](https://github.com/bazelbuild/intellij/tree/master/ijwb/ijwb.bazelproject)) to import the plugin and also that we can adjust the `--define=ij_product` to develop for a certain product version. This will help resolve issue https://github.com/bazelbuild/intellij/issues/3726

Reply:
Sure thing!